.ci.yaml

```diff
@@ -28,6 +28,7 @@ platform_properties:
         ]
       os: Linux
       device_type: none
+      clobber: "true"
```

Comment:
Unfortunately, chrome_and_driver doesn't take in clobber: true. https://cs.opensource.google/flutter/recipes/+/main:recipe_modules/flutter_deps/api.py;l=146

I think we'll want to wait for https://github.com/flutter/flutter/pull/100072 to remove the swarming cache altogether